### PR TITLE
Feat/avatar icon

### LINKED ISSUE #25: Avatar component defines styles and default avatar
# Acceptance Criteria
1. Avatar styles avatars on dashboard
2. UserListItem uses Avatar
3. If no avatar is defined, default-avatar.png displays

### PR DESCRIPTION
**Pull request by JonnoKiwi**
Closes: https://github.com/JonnoKiwi/initiative-tree/issues/25

### CODE REVIEW

**Review by johntimothybailey — CHANGES_REQUESTED** [review]
@JonnoKiwi Avatar and Icon seem to be very overlapping terms. Why `AvatarIcon` instead of just `Avatar`?

**Comment by JonnoKiwi** [comment]
> @JonnoKiwi Avatar and Icon seem to be very overlapping terms. Why `AvatarIcon` instead of just `Avatar`?

I had a conflict with using the name "avatar". I have resolved the conflict. Now the naming is more clear.

File: app/screens/Dashboard/UserListItem.tsx [hunk 1]
```diff
@@ -12,7 +10,7 @@ export default function UserListItem({ id, name, initiative, avatar, onPress })
       description={initiative}
       descriptionNumberOfLines={1}
       left={() => (
-        <Avatar.Image source={{ uri: avatar.thumbnail }} size={48} style={avatarStyle} />
+        <Avatar avatar={avatar} />
```
Comment (johntimothybailey) [inline app/screens/Dashboard/UserListItem.tsx]: `thumbail={avatar.thumbnail}`

File: app/screens/Dashboard/Avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Avatar as AvatarPaper } from 'react-native-paper'
+import { ViewStyle } from 'react-native'
+import theme from '../../theming/paper/theme'
+const defaultImage = require('./default-avatar.png')
+const avatarStyle: ViewStyle = {
+  marginRight: 24,
+}
+export default function Avatar({ avatar }) {
+  const image = (avatar === undefined) ? defaultImage : { uri: avatar.thumbnail }
```
Comment (johntimothybailey) [inline app/screens/Dashboard/Avatar.tsx]:
```
import R from 'ramda'
export default ({thumbnail}) => {
    const image = R.is(String, thumbnail) ? {  uri: thumbnail } : defaultImage
```

File: app/screens/Dashboard/Avatar.story.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import Avatar from './Avatar'
+
+declare var module
+
+storiesOf('Avatar', module)
+  .addDecorator((fn) => <StoryScreen>{fn()}</StoryScreen>)
+  .add('With default avatar', () => (
+    <Story>
+      <UseCase text="Uses default avatar" usage="The avatar property is not defined.">
+        <Avatar />
```
Comment (johntimothybailey) [inline app/screens/Dashboard/Avatar.story.tsx]: Add a case for 

```
<Avatar thumbnail={null} />
<Avatar thumbnail="" />
```

**Review by johntimothybailey — CHANGES_REQUESTED** [review]
Move it to `components/`

**Review by johntimothybailey — APPROVED** [review]

**Pull request merged**